articles/application-development/modelling/domain-designer/how-to-use-the-domain-designer/how-to-use-the-domain-designer.md

```diff
@@ -138,7 +134,7 @@ Finally, create an Association from the `Group` to the `Shape` Class and ensure
   - `Is Collection` is checked.
 - In the `Source End` section that:
   - `Is Collection` is *un*checked.
-  - `Is Nullable` is *un*checked.
```

Comment:
For the Group to Shape association to be an aggregate association then nullable should be checked on source end for the relationship to become 0..1. This is a typo as the GIF correctly shows this as the desired setting.